book/website/communication/dif-articles.md

```diff
@@ -0,0 +1,41 @@
+(cm-dif-articles)=
+# Publishing Different Article Types
+
+
+(cm-dif-articles-prerequisites)=
+## Prerequisites
+
+| Prerequisite | Importance | Skill Level | Notes |
+| -------------|----------|------|----|
+| {ref}`Open Research <rr-open>` | Helpful | Beginner | Of particular relevance is the chapter on Open Access |
+| {ref}`Data Articles<rr-rdm-article>` | Helpful | Beginner | Gives an overview of where to publish data articles |
+| {ref}`Citing Research Objects<cm-citable-cite>` | Helpful | Beginner | Information about how to cite research objects |
+
+
+(cm-dif-articles-summary)=
+## Summary
+This chapter describes the new types of articles born out of the growth of open research and the expansion in publishing all aspects of the research lifecycle. Each subchapter gives an overview of an article type.
+
+Chapters included are:
+
+* Data Papers 
+* Methods Papers
+* Micropublishing
+* Software Papers
+* Registered Reports
+
+
+(cm-dif-articles-motivation)=
+## Motivation and Background
+Traditionally journal publications have been in the format of stand alone research articles, often with a format of abstract, introduction, methods, results and discussion sections.
+
+This format was often quite limiting and meant things like datasets, software (scripts, code, apps, tools) and methods were often squeezed into research articles with little detail due to word limits, or hidden in supplementary material or not included at all. 
+
+This has led to the development of new article types, which allows all research outputs produced during the research lifecycle to be published in their own right and linked to the final interpretations and conclusions of a research project. 
+The linking of these articles is one way to fully publish a reproducible research project. 
+
+Having separate article types for these outputs allows the people involved, such as research software engineers, data stewards, to receive sufficient credit and impact for their work, particularly as publications are still the main currency by which researchers are evaluated.
```

Comment:
```suggestion
Having separate article types for these outputs allows the people involved, such as research software engineers, data stewards, to receive sufficient credit, impact and visibility for their work, particularly as publications are still the main currency by which researchers are evaluated.
```

Reply:
What about (very slight tweaks):

Having separate article types for these outputs allows the people involved, such as research software engineers and data stewards, to receive sufficient credit, attribution and visibility for their contributions to the work, particularly as publications are still the main currency by which researchers are evaluated.